pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "lm-buddy"
-version = "0.2.5"
+version = "0.2.6"
```

Comment:
related - do projects with the toml project file also have a `version.py` file somewhere?

Reply:
Not necessarily. It is possible to define a `version.py` and dynamically load that into the `pyproject.toml`, but I have chosen to avoid that for now.